test/test_torch.py

```diff
@@ -19093,6 +19093,75 @@ def verify_against_numpy(t):
                           [0, 0]], device=device, dtype=dtype)
         verify_against_numpy(t)
 
+    def _test_special_splits(self, torch_fn, np_fn, dim_range, dim, device, dtype):
```

Comment:
Can we also add some method_test entries so the autograd and jit functionality is tested, too?

Reply:
Would you also add a test (or three) to TestViewOps? It's a test suite in test_torch.py that verifies functions like this are properly returning views. @kurtamohler I actually neglected to mention this for tensor_split and we should add a test for it there, too. These tests are typically very simple. For example:

```
def test_reshape_view(self, device):
        t = torch.ones(5, 5, device=device)
        v = torch.reshape(t, (25,))
        self.assertTrue(self.is_view_of(t, v))

        v[6] = 0
        self.assertEqual(t[1, 1], v[6])
```